site-src/api-types/backendtlspolicy.md

```diff
@@ -28,6 +28,16 @@ to prevent the complications involved with sharing trust across namespace bounda
 
 All Gateway API Routes that point to a referenced Service should respect a configured BackendTLSPolicy.
 
+## Gateway Backend TLS Configuration
+
+??? example "Experimental Channel since v1.1.0"
+
+    These fields were added to Gateway in `v1.1.0`
+The Gateway specification now includes a new backendTLS field that allows configuration of TLS settings when the Gateway connects to backends. This enables specification of client certificates that the Gateway should use when establishing TLS connections with backends. The configuration includes:
+
+- [BackendTLS][backendTLS] - Defines the TLS configuration for Gateway-to-backend connections
```

Comment:
nit: Since the only field in `backendTLS` is `clientCertificateRef` then I don't think we need to list them both as separate objects.